packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -1,76 +1,93 @@
 .content {
   z-index: var(--elevation-tooltip);
   max-width: var(--popover--width);
-  max-height: 400px;
+  max-height: calc(var(--space-largest) * 8.75);
```

Comment:
where are these arbitrary numbers used for these calculations coming from??

Reply:
Based on [this comment](https://github.com/GetJobber/atlantis/pull/1530#pullrequestreview-1636377932) we need to have a max-height that shows _some_ of the next option.  There isn't a clean way to calculate this using only tokens.

Let me know if you have an alternative for the calc. I know we talked about setting 'magic numbers' but I think in this case it's a bit different.  In our `spacing.css` file and our tokens in `foundation.css` we sometimes have to calc with hard numbers.  

tldr; multiplying by 8.75 looks the best 😆

Reply:
Stylistically, maybe we could define these numbers at the top of the CSS file so it's more clear in-use? Like

`--optionsOverflowHeight: calc(var(--space-largest) * 8.75)`

and then in usage, 

`max-height: var(--optionsOverflowHeight);`

?

Reply:
Looks like we do that in components like `Button` where specific calculations are used multiple times over.

There are two instances in this css file where we use the same calculation twice.  I'm down to set those variables in a `root` at the top of the file - how do you feel about that @dlrussel?

Reply:
Diane and I talked about this over zoom. I don't want to speak for her so please @dlrussel pipe in if I get this wrong - but her concern is more so about using a hardcoded, seemingly random number in the calculation _and_ for the height.  The suggestion to use variables in a `root` won't solve for that (and I am thinking we don't need that in this file).

But we do need to have a `max-height` on the options list.  With Zak's PR now merged, a `260px` max-height looks best in the options list (final decision is with Chris).

Returning to Diane's original question, my reasoning for why the calculation and height is arbitrary is because:
- we want a slight cut off on the (fifth?) option to show the fade; afaik the easiest way to do this is through css rather than our .tsx file
- there is no single token that will get us 260px
- having `calc(var(--space-small) * 32.5)` is in my opinion, faster to calculate and easier to read than trying to divide and subtract a longer list of tokens together to somehow get to `260px`

Like I said to Diane I completely understand her hesitancy and dislike for calculations like this.  Have a think on it - if there's another way to get there let's do it